webapp/channels/src/components/admin_console/file_upload_setting.tsx

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState, createRef} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import * as Utils from 'utils/utils';
+
+import Setting from './setting';
+
+type Props = {
+    id: string;
+    label: React.ReactNode;
+    helpText?: React.ReactNode;
+    uploadingText?: React.ReactNode;
+    onSubmit: (id: string, file: File, func: (error: Error) => void) => void;
```

Comment:
this is not `ServerError` rather than `Error` right ?

Reply:
I see it used once in `schema_admin_settings.jsx` - https://github.com/mattermost/mattermost-server/blob/7029d88443973feb93eb2620330bd805c88f1cf7/webapp/channels/src/components/admin_console/schema_admin_settings.jsx#L803

I'm not 100% sure what this specific one is but after reading that i lean more to it's a string, what do you think?